pwiz_tools/Skyline/SettingsUI/FullScanSettingsControl.cs

```diff
@@ -905,8 +907,8 @@ public double TimeAroundPrediction
                 textAt.Visible = false;
                 textRes.Enabled = true;
                 textRes.Text = resCurrent.HasValue && (analyzerTypeCurrent == analyzerTypeNew)
-                                  ? resCurrent.Value.ToString(LocalizationHelper.CurrentCulture)
-                                  : TransitionFullScan.DEFAULT_CENTROIDED_PPM.ToString(LocalizationHelper.CurrentCulture);
+                                  ? resCurrent.Value.ToString(resolvingPowerFormat)
+                                  : TransitionFullScan.DEFAULT_CENTROIDED_PPM.ToString(resolvingPowerFormat);

```

Comment:
PPM values don't need this special format. They are only allowed to range between 1 and 1000, and people rarely use factional values, or if they do, I would expect them to be no finer-grained than 0.5

Reply:
OK, I'll write a single function that formats based on value type, thanks

Reply:
Sounds like a good idea.